### PR TITLE
Update 090-raw-database-access.mdx

### LINKED ISSUE #2479: Document "queryRaw doesn't support dynamic tables names"
Special case for raw queries: https://github.com/prisma/prisma/issues/9765#issuecomment-951742974

Impacted section: https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access

---

Update 8-jul-2022
Andrew: The solution for this has changed. Harshit has given me [some notes on this](https://prisma-company.slack.com/archives/D03P0UPS0QY/p1657267174671649)

### PR DESCRIPTION
**Pull request by andrew-walford-prisma**
Added caveat to $queryRaw docs that the method doesn't support dynamic PostGres table names. We link to a workaround that uses $queryRawUnsafe.

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx [hunk 1]
```diff
@@ -151,13 +155,24 @@ result.forEach((x) => {
 
 > **Note**: Prisma sends JavaScript integers to PostgreSQL as `INT8`. This might conflict with your user-defined functions that accept only `INT4` as input. If you use `$queryRaw` in conjunction with a PostgreSQL database, update the input types to `INT8`, or cast your query parameters to `INT4`.
 
+#### Dynamic table names in PostGres
```
Comment (pantharshit00) [inline content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]:
```suggestion
#### Dynamic table names in Postgres
```

File: content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx [hunk 2]
```diff
@@ -151,13 +155,24 @@ result.forEach((x) => {
...
+
+[It is not possible to send a table name as a variable](#considerations). This means that you cannot use dynamic table names with `$queryRaw`. Instead, you must use [`$queryRawUnsafe`](#queryrawunsafe), as follows:
```
Comment (pantharshit00) [inline content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]:
```suggestion
[It is not possible to interpolate table names](#considerations). This means that you cannot use dynamic table names with `$queryRaw`. Instead, you must use [`$queryRawUnsafe`](#queryrawunsafe), as follows:
```

File: content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx (hunk 1 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]: Actually, official name we use everywhere (I hope) 😆 :
```suggestion
#### Dynamic table names in PostgreSQL
```

Comment (andrew-walford-prisma) [reply content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]: Thanks Jan - I just saw this - for some reason I only saw Harshit's suggestion. Luckily, I've already made this change and merged the branch.

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]: Great to see we think alike :D

**Pull request merged**